app/src/main/java/org/oppia/app/player/exploration/ExplorationFragmentController.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.player.exploration
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.oppia.app.databinding.ExplorationFragmentBinding
+import org.oppia.app.fragment.FragmentScope
+import javax.inject.Inject
+
+/** The controller for [ExplorationFragment]. */
+@FragmentScope
+class ExplorationFragmentController @Inject constructor(
+  private val fragment: Fragment
+) {
+  fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
+    val binding = ExplorationFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
```

Comment:
Nit: we can probably shorten this to ``return ExplorationFragmentBinding.inflate(...).root``

Reply:
Resolved as per suggestion.